src/builtin/tables.scrbl

```diff
@@ -138,20 +263,19 @@ end
 
   @section{Ordering Tables}
 
-Since a table consists of a sequence of rows, one may desire to arrange
-those rows in some particular order. This can be done with any column whose
-type supports the use of @pyret{<} and @pyret{>} by using an @pyret{order}
-expression:
+To arrange the rows of a table in some particular order, use an @pyret{order}
+expression.  This can be done with any column whose
+type supports the use of @pyret{<} and @pyret{>}, including @pyret{String}s. 
```

Comment:
Tom: as of a few minutes ago, I've extended the syntax for table ordering to support multiple sort columns:
```
order some-table:
  nameX ascending,
  nameY descending,
  nameZ ascending
end
```
with as many columns as you like.  Semantically: column `nameX` is the primary sort key; in case of ties (i.e. two rows whose `nameX` values are equal), column `nameY` is used; in case that ties also, column `nameZ` is used.  The sorting is stable: If there are any duplicate rows (according to the sort criteria), they will remain in the same order as they were in the original table.

If you attempt to sort on the same column multiple times, you'll get a runtime error warning you not to do that...